src/utils/EquipmentType.js

```diff
@@ -10,6 +10,7 @@ import clsx from 'clsx';
 import React from 'react';
```

Comment:
Move this file from src/utils to src/components or src/components/ElementSearchDialog/ because its a component now not a helper function.

Reply:
done